js/samples/quickstart-nodejs/views/options.pug

```diff
@@ -0,0 +1,153 @@
+            section#sampleDisableTranslation.section
+                .immersive-reader-button.ir-button-small(onclick="handleLaunchImmersiveReader('DisableTranslation')")
+                h3#titleDisableTranslation Mountain Ranges (Translation disabled)
+                p#textDisableTranslation
+                    | The physical features of a region are often rich in resources. Within a nation, mountain ranges become natural borders for settlement areas. In the U.S., major mountain ranges are the Sierra Nevada, the Rocky Mountains, and the Appalachians.
+
+            section#sampleParent.section
+                .immersive-reader-button.ir-button-small(onclick="handleLaunchImmersiveReader('Parent')")
+                input#checkboxParent(type='checkbox' name='Parent' value='true')
+                label(for='checkboxParent')  Launch in Parent
+                br
+                h3#titleParent Kaiju (Displaying in parent)
+                p#textParent
+                    | Kaiju is a Japanese genre of films and television featuring giant monsters. The term kaiju can also refer to the giant monsters themselves, which are usually depicted attacking major cities and battling either with the military or other monsters. The kaiju genre is a subgenre of tokusatsu (special filming) entertainment.
+                #parentDiv(style='height: 400px;')
+
+script(type="text/javascript").
+
+    async function handleLaunchImmersiveReader(sampleId) {
+        getTokenAndSubdomainAsync()
+            .then(function (response) {
+                const token = response["token"];
+                const subdomain = response["subdomain"];
+
+                var langElement = $('#lang_' + sampleId);
+                var lang = sampleId == 'DisableLanguageDetection' ? langElement.val() : 'en';
+
+                const data = {
+                    title: $('#title' + sampleId).text().trim(),
+                    chunks: [{
+                        content: $('#text' + sampleId).text().trim(),
+                        lang: lang
+                    }]
+                };
+
+                let options = {};
+
+                options.disableLanguageDetection = sampleId === 'DisableLanguageDetection';
+                options.disableGrammar = sampleId === 'DisableGrammar';
+                options.disableTranslation = sampleId === 'DisableTranslation';
+                options.parent = sampleId === 'Parent' && document.getElementById("checkboxParent").checked ? document.getElementById('parentDiv') : null;
+
+                ImmersiveReader.launchAsync(token, subdomain, data, options);
+            })
+            .catch(function (error) {
+                alert("Error in getting the Immersive Reader token. Check the console.");
+                console.log(error);
```

Comment:
suggest this line come before line above so that if alert is displayed they can still check console and see the error instead of waiting until after alert is dismissed

Reply:
makes sense, done